lib/init/add-contributors-list.test.js

```diff
@@ -63,6 +60,68 @@ test('should create contributors section if content is empty', t => {
   ].join('\n');
 
   const result = addContributorsList(content);
+  t.is(result, expected);
+});
+
+test('should insert list under contributors section in Spanish', t => {
+  const content = [
+    '# project',
+    '',
+    'Description',
+    '',
+    '## Colaboradores<div id=\'contributors\'></div>',
+    ''
+  ].join('\n');
+  const expected = [
+    '# project',
+    '',
+    'Description',
+    '',
+    '## Colaboradores<div id=\'contributors\'></div>',
+    '',
+    '<!-- ALL-CONTRIBUTORS-LIST:START - Do not remove or modify this section --><!-- ALL-CONTRIBUTORS-LIST:END -->'
```

Comment:
I guess you may translate the `Do not remove or modify this section` as well, no?